src/normalize-model-and-view.js

```diff
@@ -25,6 +27,27 @@ export function getModelPath (reference) {
   return path
 }
 
+function appendModelPath (modelPath, id, internal) {
+  const addedModelPath = getModelPath(id)
+  if (internal) {
+    return `${modelPath}.properties._internal.${addedModelPath}`
+  }
+  if (modelPath === '') {
+    return addedModelPath
+  }
+  return `${modelPath}.${addedModelPath}`
+}
+
+function expandCell (cell, cellDefinitions) {
+  cell = _.clone(cell)
+  while (cell.extends) {
+    const extendedCell = cellDefinitions[cell.extends]
```

Comment:
That's a good idea. I'll open an issue for it, and for now I'll put some error throwing code that will at least making it obvious when that is the problem for now.